### PR TITLE
chore: Revise intro notebook to match demo flow

### PR DESCRIPTION
**Pull request by tssweeney**

### CODE REVIEW

File: docs/intro_notebook.ipynb [hunk 1]
```diff
@@ -51,45 +51,50 @@
...
+   "id": "0921bbef-d089-4960-b624-505172f90d76",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Log in to your W&B account\n",
-    "import weave"
+    "# Set your OpenAI API key\n",
+    "from set_env import set_env\n",
+    "\n",
+    "# Put your OPENAI_API_KEY in the secrets panel to the left 🗝️\n",
```
Comment (jamie-rasmussen) [inline docs/intro_notebook.ipynb:61]: Do we want this notebook to be colab-specific?

File: docs/intro_notebook.ipynb [hunk 2]
```diff
@@ -135,27 +138,26 @@
...
    "metadata": {},
    "outputs": [],
    "source": [
     "import weave\n",
-    "from openai import OpenAI\n",
     "\n",
     "weave.init(PROJECT)\n",
     "\n",
-    "@weave.op\n",
+    "@weave.op()\n",
```
Comment (jamie-rasmussen) [inline docs/intro_notebook.ipynb:160]: Is there a reason to put the parens in? I think we've been moving towards stripping them out.

**Review by jamie-rasmussen — APPROVED** [review]

File: docs/intro_notebook.ipynb (hunk 2 above)
Comment (tssweeney) [reply docs/intro_notebook.ipynb:160]: right now this is actually broken with code capture :/

File: docs/intro_notebook.ipynb (hunk 1 above)
Comment (tssweeney) [reply docs/intro_notebook.ipynb:61]: set_env is general purpose exactly to address this. Do agree the comment is specific.

**Pull request merged**